### ISSUE TITLE
Fix remaining issues in `tools/tts/convert-to-SE.sh.in`

### ISSUE THREAD

**Issue by snomos** [giellalt/lang-smj#22]
- [x] unknowns should just copy the input word form
- [x] when multiple variants are generated, pick the one identical or closest to the input, if no match found, pick the first one
- [x] Fix wrong handling of punctuation etc

Test data for the two cases:

```
:
"<dassta>"
        "dassta" Adv
dassta+Adv      dassta
        "dat" Pron Dem Sg Ela
dat+Pron+Dem+Sg+Ela     dassta,dasste
:
```

and

```
:
"<karstduobddágijt>"
        "karstduobddágijt" ? @X #7->0
karstduobddágijt+?      ?
:
```

I guess an AWK script would be the solution, but AWK is too unknown to me, so sending this to @flammie .

This bug is **high priority**.

**Comment by flammie** [comment]
I used a short awk script to handle the few cases here possibly :-) 
```console
$ echo dassta ja karstduobddágijt | ./convert-to-SE.sh
dassta ja karstduobddágijt
```

**Comment by snomos** [comment]
Excellent, thanks!

One issue left - a bug in the awk script that renders `PUNCT`, `CC` and `CLB` instead of the actual punctuation:

> PUNCT– AdvNávte oadtju Láhko rijkajgasskasasj stiemmpilav luonndujCLB. Álmmukmiehtse l nammadus mij le dåbdos väráldav AdvmiehtájCLB. AdvDánna válldep várajda Advsihke dajt sierralágásj karstduobddágijt CCja valjes sjaddoiellemavCLB javlaj SolhjellCLB.

**Comment by snomos** [comment]
Data before the awk script:

```
:
"<Bård>"
        "Bård" N Prop Sem/Mal Attr
Bård+N+Prop+Attr        Bård
:
"<Vegar>"
        "Vegar" N Prop Sem/Mal Attr
Vegar+N+Prop+Attr       Vegar,Vegar
:
"<Solhjellas>"
        "Solhjell" N Prop Sem/Plc Sg Ela
Solhjell+N+Prop+Sg+Ela  Solhjellas
        "Solhjell" N Prop Sem/Sur Sg Ela
Solhjell+N+Prop+Sg+Ela  Solhjellas
:
"<uddni>"
        "uddnat" <mv> V Ind Prt Sg2
uddnat+V+Ind+Prt+Sg2    uddni
"<.>"
        "." CLB
.+CLB   .
:
```

Output of the awk script:

```
:
Bård
:
Vegar
:
Solhjellas
Solhjellas
:
uddni
CLB
.
:
```

**Comment by snomos** [comment]
Everything fixed in 2fa6b42154aae06639a59957551354685b122bb5 and 0c36f5c396f7fd841fe031433606ec6f6938f047
